flink-catalog-aws/flink-catalog-aws-glue/src/main/java/org/apache/flink/table/catalog/glue/util/GlueOperator.java

```diff
@@ -0,0 +1,1359 @@
+
+        for (Expression childExpression : expression.getChildren()) {
+            if (childExpression.getChildren() != null && childExpression.getChildren().size() > 0) {
+                getExpressionString(childExpression, sb);
+            }
+        }
+        return sb.insert(
+                        0,
+                        expression.asSummaryString()
+                                + GlueCatalogConstants.SPACE
+                                + GlueCatalogConstants.AND)
+                .toString();
+    }
+
+    public Partition getGluePartition(ObjectPath tablePath, CatalogPartitionSpec partitionSpec)
+            throws PartitionNotExistException {
+        try {
+            GetPartitionRequest request =
+                    GetPartitionRequest.builder()
+                            .catalogId(getGlueCatalogId())
+                            .databaseName(tablePath.getDatabaseName())
+                            .tableName(tablePath.getObjectName())
+                            .build();
+            GetPartitionResponse response = glueClient.getPartition(request);
+            GlueUtils.validateGlueResponse(response);
+            Partition partition = response.partition();
+            if (partition.hasValues()
+                    && GlueUtils.specSubset(
+                            partitionSpec.getPartitionSpec(),
+                            partition.storageDescriptor().parameters())) {
+                return partition;
+            }
+        } catch (EntityNotFoundException e) {
+            throw new PartitionNotExistException(catalogName, tablePath, partitionSpec);
+        }
+        return null;
+    }
+
+    public boolean gluePartitionExists(ObjectPath tablePath, CatalogPartitionSpec partitionSpec)
+            throws CatalogException {
+        GetPartitionRequest request =
+                GetPartitionRequest.builder()
+                        .catalogId(getGlueCatalogId())
+                        .databaseName(tablePath.getDatabaseName())
+                        .tableName(tablePath.getObjectName())
+                        .build();
+        try {
+            GetPartitionResponse response = glueClient.getPartition(request);
+            GlueUtils.validateGlueResponse(response);
+            return response.partition()
+                    .parameters()
+                    .keySet()
+                    .containsAll(partitionSpec.getPartitionSpec().keySet());
+        } catch (EntityNotFoundException e) {
+            LOG.warn(String.format("%s is not found", partitionSpec.getPartitionSpec()));
+        } catch (GlueException e) {
+            throw new CatalogException(catalogName, e);
+        }
+        return false;
+    }
+
+    private String getGlueCatalogId() {
+        return awsConfig.getGlueCatalogId();
+    }
+
+    /**
+     * Rename glue table. Glue catalog don't support renaming table. For renaming in Flink, it has
```

Comment:
Should we throw `UnsupportedOperationException` instead for now?
Also Please add Jira to track TODO items.